.circleci/config.yml

```diff
@@ -0,0 +1,240 @@
+version: 2.1
+references:
+  pre_checkout: &pre_checkout
+    run:
+        name: Pre Checkout
+        command: |
+          stat -c '%y' /opt/ros/$ROS_DISTRO/setup.sh | \
+            sha256sum | \
+            awk '{print "distro "$1}' > /opt/ros/$ROS_DISTRO/checksum.txt
+          cp $MOVEIT2_WS/src/moveit2/tools/ros2_dependencies.repos $ROS_WS/
```

Comment:
there's already a repos file from @LanderU, but it hasn't been added to ``ros-planning`` yet:

https://github.com/AcutronicRobotics/moveit2/blob/master/moveit2.repos